### PR TITLE
fix(RHCLOUD-24793): Show ROS tab is azure or aws cloud provider

### PR DESCRIPTION
**Pull request by karelhala**
### Description

ROS team brought to our attention that their tab is always hidden on inventory detail. This happend because of recent inventory refactoring and we forgot to move certain part of the code. This PR re-enables it and also removes unused part of redux code which was used in the past to calculate the visibility.

### Regression

https://github.com/RedHatInsights/insights-inventory-frontend/pull/1435

### CODE REVIEW

**Comment by karelhala** [comment]
restarting jenkins

File: src/routes/InventoryDetail.js [hunk 1]
```diff
@@ -75,6 +75,11 @@ const Inventory = () => {
     const writePermissions = useWritePermissions();
     const entityLoaded = useSelector(({ entityDetails }) => entityDetails?.loaded);
     const entity = useSelector(({ entityDetails }) => entityDetails?.entity);
+    const cloudProvider = useSelector(({ systemProfileStore }) => systemProfileStore?.systemProfile?.cloud_provider);
+    const availableApps = useMemo(() => appList.map((app) => app.name === 'ros' ? {
+        ...app,
+        isVisible: ['aws', 'azure'].includes(cloudProvider)
```
Comment (PreetiW) [inline src/routes/InventoryDetail.js:81]: @karelhala ATM ros is only supported for aws, can you please update this accordingly!

**Review by mkholjuraev — APPROVED** [review]
Just a minor suggestion.

**Comment by karelhala** [comment]
@mkholjuraev the file should be in right place. Anything else to adjust? Thank you.

**Comment by mkholjuraev** [comment]
@karelhala thank you for letting me know. Nope, all seems good!

**Pull request merged**